### ISSUE TITLE
Obtain IPCW beyond max train time

### ISSUE THREAD

**Issue by melodiemonod** [Novartis/torchsurv#102]
We cannot obtain the IPCW on the test set for observations with `time` greater than the max `time` in the training set.

**Comment by melodiemonod** [comment]
Sorry, I haven’t had time to work on this. I will try to do it within the next month.
